README.md

```diff
@@ -0,0 +1,147 @@
+# Stackable Docker Images
+
+This repository contains Dockerfiles and scripts to build base images for use within Stackable.
+
+<!-- start:badges: autogenerated by ./update_readme_badges.sh -->
+| | | | |
+| -: | -: | -: | -: |
+| [![Build Airflow]][dev_airflow.yaml] | [![Build Druid]][dev_druid.yaml] | [![Build Hadoop]][dev_hadoop.yaml] | [![Build HBase]][dev_hbase.yaml] |
+| [![Build Hello-World]][dev_hello-world.yaml] | [![Build Hive]][dev_hive.yaml] | [![Build Java Base]][dev_java-base.yaml] | [![Build Java Development]][dev_java-devel.yaml] |
+| [![Build Kafka Testing Tools]][dev_kafka-testing-tools.yaml] | [![Build Kafka]][dev_kafka.yaml] | [![Build kcat]][dev_kcat.yaml] | [![Build Krb5]][dev_krb5.yaml] |
+| [![Build NiFi]][dev_nifi.yaml] | [![Build Omid]][dev_omid.yaml] | [![Build OPA]][dev_opa.yaml] | [![Build Spark K8s]][dev_spark-k8s.yaml] |
+| [![Build Stackable Base]][dev_stackable-base.yaml] | [![Build Superset]][dev_superset.yaml] | [![Build Testing Tools]][dev_testing-tools.yaml] | [![Build Tools]][dev_tools.yaml] |
+| [![Build Trino CLI]][dev_trino-cli.yaml] | [![Build Trino]][dev_trino.yaml] | [![Build Vector]][dev_vector.yaml] | [![Build ZooKeeper]][dev_zookeeper.yaml] |
+<!-- end:badges -->
+
+## Prerequisites
+
+* Stackable Image Tools (`pip install image-tools-stackabletech`) https://github.com/stackabletech/image-tools
```

Comment:
note: The `markdownlint` action complains about the bare link here and below. I guess we should fix that.

Reply:
I'll go through and resolve the lint findings.

Reply:
Fixed in 6b1e799 and 026ed45.
Also added bash strict mode in 72b8bf3.

Reply:
Looking good!